### ISSUE TITLE
Tracking Issue for 'ordering helpers'

### ISSUE THREAD

**Issue by jnqnfe** [rust-lang/rust#79885]
Feature gate: `#![feature(ordering_helpers)]`

This is a tracking issue for the 'ordering helpers' feature, which sets out to add some simple helpers to `Ordering` such as `is_ge()` for checking whether an instance of `Ordering` is 'greater than or equal' (is the `Greater` or `Equal` variant, not the `Less` variant).

### Public API

```rust
// core::cmp
impl Ordering {
    pub const fn is_eq(self) -> bool;
    pub const fn is_ne(self) -> bool;
    pub const fn is_lt(self) -> bool;
    pub const fn is_gt(self) -> bool;
    pub const fn is_le(self) -> bool;
    pub const fn is_ge(self) -> bool;
}
```

### Steps / History

- [x] Implementation: https://github.com/rust-lang/rust/pull/79656
- [x] Final commenting period (FCP)
- [x] Stabilization PR: https://github.com/rust-lang/rust/issues/79885

### Unresolved Questions

- None yet.

**Comment by jnqnfe** [comment]
@scottmcm / @sfackler Please advise on next steps.

 - Do I make the stabilization PR myself or a team member?
 - Given this is such a trivial addition, how much time roughly to wait before doing so? (I presume the FCP starts after that PR is created?)

**Comment by m-ou-se** [comment]
@jnqnfe Thanks for working on this! I've edited this tracking issue to match the new template for library tracking issues we have now. (It didn't exist yet when you opened this issue. But from now on there should be a 'library tracking issue' option when filing a new issue.)

> Do I make the stabilization PR myself or a team member?

To make sure the discussion doesn't split over two different issues/PRs, we prefer to keep everthing including the stabilization decision on this tracking issue. A libs team member can kick off the stabilization process, at which point the rfcbot will ask all the team members to verify they agree with stabilization. Once enough members agree and there are no concerns, the 'final commenting period' begins: this issue will be marked as such and will be listed in the next This Week in Rust newsletter. If no blocking concerns are raised in that period of 10 days, a stabilzation PR can be opened by anyone which can then be approved right away.

> Given this is such a trivial addition, how much time roughly to wait before doing so?

Depends a bit on how much in a hurry you are with this new feature, but it's usually a good idea to wait a few release cycles. That way, it has also been visible in the stable documentation for a while (marked as nightly-only), giving it some more exposure to the world.

> I presume the FCP starts after that PR is created?

We recently decided to change that, to avoid a new discussion starting on that PR separate from the discussion on the tracking issue. We now prefer to start the FCP on the tracking issue first. (I'll update the new tracking issue template to include this information.)
